### ISSUE TITLE
SNES EJECT button support

### ISSUE THREAD

**Issue by petrockblog** [petrockblog/ControlBlockService2#32]
Currently I'm working on my second RaspberryPi/RetropPie inside a SNES case project. When I started my first project was at the time when support for the SNES reset button was added to the ControlBlock SNES gamepad driver. I really liked that feature as it provides a convenient to exit a running game, especially helpful for casual games who resist to learn hotkeys to do so. I always had the idea to also make use of the SNES eject button. In my setup I'd like the reset button to restart the currently running game (Select+B) and the eject button exit the game (Select+Start).

Long story short: is there any chance that this is integrated into the ControlBlock SNES driver? Or is there already a generic way to configured custom keys/switches connected to the ControlBlock?
